2018-07-15 12:11:12	 Test UDP Tunnel
2018-07-15 12:11:12	 ===============
wlan0
2018-07-15 12:11:12	 configure device
2018-07-15 12:11:12	 enabled telnet ........
Traceback (most recent call last):
  File "/home/lantronix/MIDAS/scripts/tunneludp.py_acm", line 292, in <module>
    device_config()
  File "/home/lantronix/MIDAS/scripts/tunneludp.py_acm", line 186, in device_config
    clientid.expect('status Device')
  File "/usr/lib/python2.7/dist-packages/pexpect/spawnbase.py", line 315, in expect
    timeout, searchwindowsize, async)
  File "/usr/lib/python2.7/dist-packages/pexpect/spawnbase.py", line 339, in expect_list
    return exp.expect_loop(timeout)
  File "/usr/lib/python2.7/dist-packages/pexpect/expect.py", line 102, in expect_loop
    return self.eof(e)
  File "/usr/lib/python2.7/dist-packages/pexpect/expect.py", line 49, in eof
    raise EOF(msg)
pexpect.exceptions.EOF: End Of File (EOF). Exception style platform.
<pexpect.pty_spawn.spawn object at 0x7f70dfc79c50>
command: /usr/bin/telnet
args: ['/usr/bin/telnet', '192.168.51.105']
searcher: None
buffer (last 100 chars): ''
before (last 100 chars): '\r\n\r\ndevice\r\n\r\ntelnet: Unable to connect to remote host: No route to host\r\n'
after: <class 'pexpect.exceptions.EOF'>
match: None
match_index: None
exitstatus: 1
flag_eof: True
pid: 3070
child_fd: 5
closed: False
timeout: 30
delimiter: <class 'pexpect.exceptions.EOF'>
logfile: None
logfile_read: None
logfile_send: None
maxread: 2000
ignorecase: False
searchwindowsize: None
delaybeforesend: 0.05
delayafterclose: 0.1
delayafterterminate: 0.1
